packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -172,9 +172,14 @@ library LibProving {
             inputs[8] = (uint256(uint160(evidence.prover)) << 96)
                 | (uint256(evidence.parentGasUsed) << 64) | (uint256(evidence.gasUsed) << 32);
 
+            // Also hash configs that will be used by circuits
```

Comment:
@davidtaikocha  please take a look at this.

Reply:
yah i believe they are also necessary. @smtmfft @johntaiko

Reply:
Is this change will in a3 testnet?

Reply:
Can we remove the `parentHash` and `gasUsed`, because these are contained in the `blockHash`?

Reply:
We need `parentHash` to make sure the blocks are correctly linked with each other. `gasUsed` we need in the smart contract, it's not directly available in the block hash as far as I know.